packages/stitch/src/createMergedTypeResolver.ts

```diff
@@ -19,7 +19,7 @@ export function createMergedTypeResolver<TContext extends Record<string, any> =
       subschema,
       selectionSet,
       key,
-      type = getNamedType(info.returnType) as GraphQLOutputType,
+      type = getNamedType(info?.returnType) as GraphQLOutputType,
```

Comment:
Avoid casts. Since you're optionally chaining info.returnType, the result may not be `GraphQLOutputType`. If you're 100% sure it'll be that type, explain why and use a null assertion instead.

Reply:
This change is irrelevant though. I'll remove it